clang/test/dpct/query_api_mapping/Thrust/thrust_api_test.cu

```diff
@@ -0,0 +1,276 @@
+// UNSUPPORTED: cuda-8.0
+// UNSUPPORTED: v8.0
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=thrust::set_union_by_key | FileCheck %s -check-prefix=set_union_by_key
```

Comment:
Please add -stop-on-parse-err option for the lit case.

Reply:
okay